src/sbml/SBase.cpp

```diff
@@ -7288,20 +7053,17 @@ SBase::updateSBMLNamespace(const std::string& package, unsigned int level,
         uri = SBML_XMLNS_L2V4;
         break;
       case 5:
-      default:
```

Comment:
Can you remember why removing dafults here was good ?
No problem if you can't.

Reply:
My rationale here was that it doesn't make sense for the default case to *actually* mean case 5, so the containing code should go in case 5. Sonarcloud was complaining about the empty `case` anyway. However, sonarcloud actually doesn't like switch-case blocks without defaults  (as I've left this) but I don't really know what the default *should* be in this case (I would assume an error message?) so I left the default case out so it remains flagged in sonarcloud to be fixed in the future.

I realise this is a bit complicated so let me know if you think it should be reverted.

Reply:
Yes that one should be reverted as the default do correspond to the default levels and versions of sbml.